data/102/527/513/102527513.geojson

```diff
@@ -182,9 +182,10 @@
     ],
     "oa:elevation_ft":"13",
     "oa:type":"large_airport",
-    "src:geom":"uszcta",
+    "src:geom":"ccsf-sfo",
     "src:geom_alt":[
-        "woedb"
+        "woedb",
+        "uszcta"
```

Comment:
@thisisaaronland possible to add "uszcta" to the `"wof:geom_alt"`property as well? Other than that, this PR looks good.